src/main/java/uk/gov/hmcts/cmc/claimstore/services/document/AlwaysGenerateDocumentsService.java

```diff
@@ -15,24 +18,37 @@
 public class AlwaysGenerateDocumentsService implements DocumentsService {
 
     private final ClaimService claimService;
+    private final ClaimIssueReceiptService claimIssueReceiptService;
     private final LegalSealedClaimPdfService legalSealedClaimPdfService;
     private final DefendantResponseCopyService defendantResponseCopyService;
+    private final DefendantResponseReceiptService defendantResponseReceiptService;
     private final CountyCourtJudgmentPdfService countyCourtJudgmentPdfService;
     private final SettlementAgreementCopyService settlementAgreementCopyService;
+    private final DocumentTemplates documentTemplates;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "documentTemplates" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1068)